src/core/api/search.go

```diff
@@ -66,11 +67,11 @@ func (s *SearchAPI) Get() {
 			s.ParseAndHandleError("failed to get projects", err)
 			return
 		}
-		if isAuthenticated {
-			mys, err := s.SecurityCtx.GetMyProjects()
+		if sc, ok := s.SecurityCtx.(*local.SecurityContext); ok {
```

Comment:
Is this intentional? Previously it doesn't require the securityContext must be local based.